src/simdvec.jl

```diff
@@ -448,7 +448,7 @@ const HORZ_REDUCTION_OPS = [
     (min , IntTypes      , Intrinsics.reduce_smin)
     (min , UIntTypes     , Intrinsics.reduce_umin)
     (min , FloatingTypes , Intrinsics.reduce_fmin)
-    (+   , IntegerTypes  , Intrinsics.reduce_add)
+    (+   , Union{IntegerTypes, Bool}  , Intrinsics.reduce_add)
```

Comment:
Won't reducing e.g. `<true, true, false, false>` lead to the bitpattern `00000010` which is an invalid pattern for a Julia bool so calling that will likely be undefined behavior?

Reply:
Oh, I see the newly added `reduce_add` now.